src/cow-react/modules/limitOrders/pure/ReceiptModal/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { GpModal } from 'components/Modal'
+import { CurrencyAmount, Fraction, Token } from '@uniswap/sdk-core'
+import * as styledEl from './styled'
+import { OrderKind } from 'state/orders/actions'
+import { CloseIcon } from 'theme'
+import { CurrencyField } from './CurrencyField'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { ParsedOrder } from '@cow/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList'
+import { FeeField } from './FeeField'
+import { StyledScrollarea } from 'components/SearchModal/CommonBases/CommonBasesMod'
+import { FieldLabel } from './FieldLabel'
+import { PriceField } from './PriceField'
+import { DateField } from './DateField'
+import { FilledField } from './FilledField'
+import { SurplusField } from './SurplusField'
+import { OrderIDField } from './OrderIdField'
+import { StatusField } from './StatusField'
+
+interface ReceiptProps {
+  isOpen: boolean
+  order: ParsedOrder
+  chainId: SupportedChainId
+  onDismiss: () => void
+  sellAmount: CurrencyAmount<Token>
+  buyAmount: CurrencyAmount<Token>
+  limitPrice: Fraction | null
+  executionPrice: Fraction | null
+}
+
+enum Tooltip {
+  STATUS = 'Status info TODO',
+  LIMIT_PRICE = 'Limit price TODO',
+  EXECUTION_PRICE = 'Execution price TODO',
+  FILLED = 'Filled TODO',
+  SURPLUS = 'Order Surplus TODO',
+  FEE = 'Fee TODO',
+  CREATED = 'Created date TODO',
+  EXPIRY = 'Expiry date TODO',
+  ORDER_TYPE = 'Order type TODO',
+  ORDER_ID = 'Order id TODO',
+}
+
+export function ReceiptModal({
+  isOpen,
+  onDismiss,
+  order,
+  chainId,
+  sellAmount,
+  buyAmount,
+  limitPrice,
+  executionPrice,
```

Comment:
ubernit, and probably personal preference, but with so many params, i prefer to move the destructuring to the implementation (not in the funciton definition).

I like to have simple function definitions because sometimes you just want to quicly get an idea of the signatures (read the name, param type, and return type) and you don't want to go over too many lines to do that. 

Having it in the body makes it very easy also to see the props used, if you want to review that after checking the signature.